docs/development/reviewing-patches.rst

```diff
@@ -36,6 +36,9 @@ These are small things that are not caught by the automated style checkers.
 * Does a variable need a better name?
 * Should this be a keyword argument?
 
+When using ``target="_blank_"`` for a hyperlink (usually to an external site),
```

Comment:
Small nit, but this point falls in the "implementation" review tier, not "grammar and style". This is because using `rel="nooper"` is a function change -- ensuring that the foreign website doesn't get a `windows.opener` that can be used to manipulate PyPI -- not a stylistic, code readability one.

Reply:
Thank you for your feedback, @alex. I shall move `rel="noopener"` to the "implementation" review tier. May I also know whether the sentence about `rel="noopener"` be added to the existing list or as a separate sentence in a new paragraph?

Reply:
@pgadige I think we can add this as an additional `li` to the list, to be consistent with the rest of the items.  Thanks!